docs/user_guide/15_Writing_data.md

```diff
@@ -16,13 +16,17 @@ from deepforest import get_data
 import rasterio as rio
 import geopandas as gpd
 from matplotlib import pyplot as plt
+from shapely import geometry
+import os
 
 annotations = get_data("2018_SJER_3_252000_4107000_image_477.csv")
 path_to_raster = get_data("2018_SJER_3_252000_4107000_image_477.tif")
 src = rio.open(path_to_raster)
 original = utilities.read_file(annotations)
 
-geo_coords = utilities.image_to_geo_coordinates(original, root_dir=os.path.dirname(path_to_raster))
+raster_path = get_data("OSBS_029.tif")  # Change with your raster path
+
+geo_coords = utilities.image_to_geo_coordinates(original, root_dir=os.path.dirname(raster_path))
```

Comment:
This change doesn't align with the code above because `annotations` and therefore `original` is from the SJER dataset. Seems like the simple change here is just is the just change `raster_path` in the `geo_coords` line to `path_to_raster`. If you want to change the `path_to_raster` name in both places to `raster_path` I think that would be fine too.